docs/api.md

```diff
@@ -885,6 +885,52 @@ Check if all the services are valid and can be deployed.
 }
 ```
 
+### `GET /api/v2/services/deployment`
+
+Get all services deployment information.
+
+**Response (Success - 200):**
+
+```json
+{
+  "service_config_id1": {
+    "status": 3,  // DEPLOYED
```

Comment:
can we improve the serialization and deserialization of the deployment status to a more readable state, like DEPLOYED, STOPPED, etc.?

Reply:
We can do this in a backward compatible fashion also, by keeping the integer deserializing but only serializing into strings.